### PR TITLE
Fumo item

### PR DESCRIPTION
**Pull request by Govorunb**
Fixes the game-breaking issue where you couldn't hold and hug the fumo
I also applied transforms on the fbx because it was bothering me

There are still minor issues with near plane clipping in BZ (and the thing with IK rotating on draw) but they can just go on the backlog :Clueless:

### CODE REVIEW

File: Unity/Assets/Scripts/SCHIZO/Items/FumoItem/FumoItemLoader.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#if UNITY_EDITOR
+namespace SCHIZO.Items.Data
+{
+    public partial class ItemData
+    {
+        [ContextMenu("Set Loader/Fumo Item")]
+        private void CreateFumoItemLoader() => AssignItemLoader(CreateInstance<FumoItem.FumoItemLoader>());
+    }
+}
+#endif
```
Comment (Alexejhero) [inline Unity/Assets/Scripts/SCHIZO/Items/FumoItem/FumoItemLoader.cs]: move to actual ItemData class pls

File: SCHIZO/Items/Gymbag/GymbagLoader.cs [hunk 2]
```diff
@@ -1,9 +1,11 @@
+using SCHIZO.Items.Data;
+
 namespace SCHIZO.Items.Gymbag;
 
 partial class GymbagLoader
 {
     public override void Load()
     {
-        new Gymbag(itemData.ModItem, itemData.CloneTarget).Register();
+        new Gymbag(itemData.ModItem, ((CloneItemData)itemData).CloneTarget).Register();
```
Comment (Alexejhero) [inline SCHIZO/Items/Gymbag/GymbagLoader.cs:9]: blabbering

File: SCHIZO/Items/FumoItem/FumoItemLoader.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+using UnityEngine;
+
+namespace SCHIZO.Items.FumoItem;
+
+partial class FumoItemLoader
+{
+    public override void Load()
+    {
+        new FumoItem(itemData.ModItem).Register();
+        // TODO: move to a "coordinated spawns" scriptable object?
```
Comment (Alexejhero) [inline SCHIZO/Items/FumoItem/FumoItemLoader.cs:12]: yes

File: SCHIZO/Items/FumoItem/FumoItem.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    protected override void PostRegister()
+    {
+#if BELOWZERO
+        Nautilus.Handlers.CraftDataHandler.SetColdResistance(ModItem, 20);
+#endif
+        base.PostRegister();
+    }
+}
```
Comment (Alexejhero) [inline SCHIZO/Items/FumoItem/FumoItem.cs]: move to unity

**Review by Alexejhero — CHANGES_REQUESTED** [review]

**Comment by Govorunb** [comment]
i've added project items based on the comments 👍

**Review by Alexejhero — APPROVED** [review]

**Pull request merged**